<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anim_tiles_edit</class>
 <widget class="QWidget" name="anim_tiles_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Current Block:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Graphic:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addBlock_button">
         <property name="text">
          <string>Add Block</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Frame Delay:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="currentBlock_combo"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="graphic_combo"/>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="frameDelay_spinBox">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Import Pack</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="framesPreviewArea" name="animTileFrames_widget" native="true">
         <property name="styleSheet">
          <string notr="true">border-width: 2px;
border-color: black;
border-style: outset;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Frames:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="animTitle" name="animTilePreview_widget" native="true">
         <property name="styleSheet">
          <string notr="true">border-width: 2px;
border-color: black;
border-style: outset;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Preview:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>framesPreviewArea</class>
   <extends>QWidget</extends>
   <header>framespreviewarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>animTitle</class>
   <extends>QWidget</extends>
   <header>animtitle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
